mos_tests/object_storage/test_crud_operations.py

```diff
@@ -0,0 +1,304 @@
+        f_path, f_name = create_file_on_node
+        bname, _ = s3cmd_create_container
+        s3cmd_client.bucket_put_file(bname, f_path)
+        s3cmd_client.bucket_del_file(bname, f_name)
+        ls_in = s3cmd_client.bucket_ls(bname)
+        s3cmd_client.bucket_remove(bname, recursive=True)
+        obj_path = '{0}/{1}'.format(bname, f_name)
+        assert obj_path not in ''.join(ls_in)
+
+    @pytest.mark.testrail_id('842479')
+    @pytest.mark.parametrize('create_file_on_node', [f_size_b], indirect=True)
+    def test_object_storage_s3cmd_obj_upload_big(
+            self, s3cmd_client, s3cmd_create_container, create_file_on_node):
+        """Upload big object to Object Storage (RadosGW)
+
+        Actions:
+        1. Create file (object) on controller.
+        2. Create container via s3cmd cli utility.
+        3. Put object to container with option 'multipart-chunk-size-mb'.
+        4. With list check that object present in container.
+        5. Check that size of container is the same as it was created.
+        6. Check expected and actual chunk slice number.
+        5. Delete container.
+        """
+        file_size_b = str(self.f_size_b * 1024 * 1024)
+        f_path, f_name = create_file_on_node
+        bname, _ = s3cmd_create_container
+
+        put_out = s3cmd_client.bucket_put_file(
+            bname, f_path, chunk=self.chunk_size)
+        ls_in = s3cmd_client.bucket_ls(bname)
+        s3cmd_client.bucket_remove(bname, recursive=True)
+        obj_path = '{0}/{1}'.format(bname, f_name)
+        # check object present in ls
+        assert obj_path in ''.join(ls_in)
+        # check size of obj is correct
+        assert file_size_b in ''.join(ls_in)
+        # check num of slices is correct
+        exp_chunk_slices = math.ceil(
+            float(self.f_size_b) / float(self.chunk_size))
+        # MBs left in last slice
+        mbleft = int(
+            self.f_size_b - (self.chunk_size * (exp_chunk_slices - 1)))
+        exp_string = (
+            "upload:.+{f_path}.+->.+s3://{obj_path}.+"
+            "[part {part} of {part}, {mbleft}MB]"
+        ).format(f_path=f_path, obj_path=obj_path, part=int(exp_chunk_slices),
+                 mbleft=mbleft)
+        # check string present in output
+        assert re.match(exp_string, put_out) is not None
+        # Example of search string for re.match (exp_string):
+        #   upload: '/root/s3cmd_test_obj' ->
+        #   's3://TESTBUCKET1772/s3cmd_test_obj'  [part 4 of 4, 138MB] [1 of 1]
+
+
+@pytest.mark.undestructive
+@pytest.mark.check_env_('is_ceph_enabled')
+@pytest.mark.check_env_('is_radosgw_enabled')
+class TestObjectStorageSWIFT(TestBase, Helpers):
+    """Object Storage - CRUD Operations Tests
+    """
+    f_size = 100         # MB
+    f_size_b = 5432      # MB
+    segment_size = 1024  # MB
+
+    # Set "autouse=True" only for test-automation purposes
+    @classmethod
+    @pytest.fixture(scope='class', autouse=False)  # True to enable
```

Comment:
To tell the truth, I don't know.
There is a slight possibility that there will be a need to make changes or add something.